### ISSUE TITLE
Setting nginx.replicas to 0 doesn't seem to work

### ISSUE THREAD

**Issue by jbialy** [goharbor/harbor-helm#322]
I'm trying to install Harbor with a minimal amount of components. Here's the helm command that I'm using:

```
helm install --name image-cache harbor/harbor --set notary.enabled=false --set chartmuseum.enabled=false --set clair.enabled=false --set expose.type=clusterIP --set expose.tls.enabled=false --set persistence.enabled=false --set nginx.replicas=0 --set portal.replicas=0
```

The above results in the portal deployment being created with the replica count set to zero  which is exactly what I want. However, `--set nginx.replicas=0` seems to have no affect on the nginx deployment that gets created, the nginx replica count is still set as 1.

```
$ k get pod
NAME                                             READY   STATUS             RESTARTS   AGE
image-cache-harbor-core-75587ccd4b-5bwwr         1/1     Running            2          3m58s
image-cache-harbor-database-0                    1/1     Running            0          3m58s
image-cache-harbor-jobservice-6ddb9f8b48-hmmnx   1/1     Running            1          3m58s
image-cache-harbor-nginx-555cb585c7-b5j8z        0/1     CrashLoopBackOff   5          3m58s
image-cache-harbor-redis-0                       1/1     Running            0          3m57s
image-cache-harbor-registry-7768b4cddb-mdgf2     2/2     Running            0          3m58s
```

```
$ k describe deployment images-harbor-nginx
...
Replicas:               1 desired | 1 updated | 1 total | 0 available | 1 unavailable
...
```

```
$ k describe deployment image-harbor-portal
Replicas:               0 desired | 0 updated | 0 total | 0 available | 0 unavailable
```

My helm / tiller versions:

```
Client: &version.Version{SemVer:"v2.13.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.13.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
```

**Comment by ywk253100** [comment]
It's a known issue and has been fixed by https://github.com/goharbor/harbor-helm/pull/270 in master branch. The new version of Harbor chart will be available when the Harbor 1.9 comes out

**Comment by jbialy** [comment]
@ywk253100 Thanks! Is there any way that the Harbor Chart version could get bumped independently of the Harbor version? I'm thinking that it might be nice to get the Chart fixes released so that they are available in the harbor helm repo!?

**Comment by ywk253100** [comment]
@jbialy The version of Harbor chart is already independent with Harbor version now. We just released Harbor chart 1.1.2 5 days ago. 
As the Harbor 1.9 will release soon, maybe late August, we'll not release a Harbor chart unless there are some breaking bugs

**Comment by jbialy** [comment]
> @jbialy The version of Harbor chart is already independent with Harbor version now. We just released Harbor chart 1.1.2 5 days ago.

That's great!

> As the Harbor 1.9 will release soon, maybe late August, we'll not release a Harbor chart unless there are some breaking bugs

I'm just wondering as I think it might be a good idea to have incremental changes to the chart independent of a major Harbor release? In my case, I'd love to get the option to scale down the nginx replica count to zero. Given that the change is already in master, simply doing a minor bump to the Chart version and merging a few of these features would be really helpful!
